README.rst

```diff
@@ -95,8 +94,7 @@ See https://docs.scrapy.org/en/master/contributing.html for details.
 Code of Conduct
 ---------------
 
-Please note that this project is released with a Contributor Code of Conduct
-(see https://github.com/scrapy/scrapy/blob/master/CODE_OF_CONDUCT.md).
+Please note that this project is released with a Contributor `Code of Conduct <./CODE_OF_CONDUCT.md>`_.
```

Comment:
We're using README as long_description for pypi; I'm not sure the relative link would work there.